agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1669,10 +1637,29 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
     }
 
     switch (beerocks_header->action_op()) {
-    case beerocks_message::ACTION_APMANAGER_INIT_DONE_NOTIFICATION: {
-        LOG(INFO) << "received ACTION_APMANAGER_INIT_DONE_NOTIFICATION from sd=" << intptr_t(sd);
+    case beerocks_message::ACTION_APMANAGER_UP_NOTIFICATION: {
+        LOG(INFO) << "received ACTION_APMANAGER_UP_NOTIFICATION from sd=" << intptr_t(sd);
+        if (ap_manager_socket) {
+            LOG(ERROR) << "AP manager opened new socket altough the exist socket is already open!";
```

Comment:
```suggestion
            LOG(ERROR) << "AP manager opened new socket altough the existing socket is still open!";
```

Reply:
Also, not sure it's an ERROR (maybe a WARNING), as we simply close the old socket and continue...

Reply:
The thing is, that is should not occur at all. 
Changed the print.